test/specs/GetTextSpec.ts

```diff
@@ -1,29 +1,23 @@
 import { assert, expect } from 'chai';
 import { BrowserUtils } from '../../src/commons/BrowserUtils';
 import { describeCommon } from '../TestHelper';
+import getText = BrowserUtils.getText;
```

Comment:
I wouldn't do this in a reference test. It can confuse readers that missed this line

Reply:
fixed